runtime/autoload/health/provider.vim

```diff
@@ -45,6 +45,19 @@ function! s:download(url) abort
   return 'missing `curl` and `python`, cannot make pypi request'
 endfunction
 
+" Check for clipboard tools.
+function! s:check_clipboard() abort
+  call health#report_start('Clipboard')
+
+  let clipboard_tool = provider#clipboard#Executable()
+  if empty(clipboard_tool)
+    call health#report_warn(
+          \ "No clipboard tool found. Using the system clipboard won't work.",
+          \ ['See ":h clipboard"'])
```

Comment:
should spell out `:help`.

Future idea: highlight help tags in `:CheckHealth` and make `K` work on them.